content/hi/container.md

```diff
@@ -0,0 +1,27 @@
+---
+शीर्षक: कंटेनर
+स्थिति: Completed
+श्रेणी: टेक्नोलॉजी
+---
+
+### यह क्या है
+--- 
+    
+एक कंटेनर एक चलने वाली एक प्रक्रिया है जिसमें कंप्यूटर के ऑपरेटिंग सिस्टम द्वारा प्रबंधित संसाधन और क्षमता की कमी होती है। कंटेनर प्रक्रिया के लिए उपलब्ध फ़ाइलें एक कंटेनर छवि के रूप में पैक की जाती हैं।
```

Comment:
```suggestion
कंटेनर एक चलने वाली प्रक्रिया है जिसके संसाधन और क्षमता पर प्रतिबंध कंप्यूटर के ऑपरेटिंग सिस्टम द्वारा प्रबंधित होती है। कंटेनर प्रक्रिया के लिए उपलब्ध फ़ाइलें एक कंटेनर छवि के रूप में पैक की जाती हैं। 
```
- eliminating multiple usage of एक makes the sentence easier to read and understand. 
- rephrasing capability constraints as क्षमता पर प्रतिबंध instead of क्षमता की कमी